pontoon/base/middleware.py

```diff
@@ -48,3 +48,19 @@ def process_request(self, request):
             if user:
                 request.user = user
                 auth.login(request, user)
+
+
+class CookiesSameSiteStrict(object):
+    """
+    SameSite attribute is the new security mechanism that should improve protection against CSRF attacks.
+    Support for SameSite attribute in Cookies is implemented in Django 2.0 and won't be backported to Django 1.11.x
+    This middleware will be obsolete when Pontoon will be ported to Django 2.0 and should as a follow-up action.
```

Comment:
Looks like a verb is missing at the end of the last sentence.

Also, please make lines shorter than 79 characters.